stuttgart-media-university-author-date.csl

```diff
@@ -0,0 +1,220 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" default-locale="de-DE" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Hochschule der Medien Stuttgart (author-date; German)</title>
+    <title-short>HdM Stuttgart (author-date)</title-short>
+    <id>http://www.zotero.org/styles/stuttgart-media-university-author-date</id>
+    <link href="http://www.zotero.org/styles/stuttgart-media-university-author-date" rel="self"/>
+    <link href="http://www.zotero.org/styles/stuttgart-media-university" rel="template"/>
+    <link href="https://github.com/citation-style-language/styles/issues/175" rel="documentation"/>
+    <link href="http://www.hdm-stuttgart.de/" rel="documentation"/>
+    <author>
+      <name>Brenton M. Wiernik</name>
+      <email>zotero@wiernik.org</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="social_science"/>
+    <updated>2012-09-27T22:06:38+00:00</updated>
```

Comment:
(slightly outdated)

Reply:
What do you mean here?

Reply:
While we don't strictly enforce correct 'updated' timestamps on style submission, it would still be good if this didn't refer to 2012 as it looks you edited this style very recently.

(Our release branch will have a more accurate time stamp based on the git commit date in "master")

Reply:
Oh okay, I've just never changed update times because I've assumed they updated automatically.

Reply:
I think `<updated/>` fulfills two roles. In the release branch, it is an accurate indicator of when the style in our repository was last updated (as e.g. displayed to end users at https://www.zotero.org/styles). In "master", it can be a (rough) indicator of when a style was last edited by the contributor (which can differ significantly from the eventual git modification date as sometimes people don't submit styles for years after they create them, or in case of languishing PRs).

Anyway, I typically manually set mine to the date I edit/submit the style (and sometimes tweak PRs if they're very far out of whack, like https://github.com/citation-style-language/styles/pull/5138/commits/b490ab01e410714841a35a02e0055eb859d1611a in https://github.com/citation-style-language/styles/pull/5138 ).